### PR TITLE
Add brand assets page

### LINKED ISSUE #2953: Brand Refresh – Review and publish "Distinctive Brand Assets" page
Review and merge this PR which provides code examples of how to easily implement all of the added brand assets. Includes links back to Brand Central guidance as well.

PR: https://github.com/grommet/hpe-design-system/pull/2951

### PR DESCRIPTION
**Pull request by halocline**
[Deploy Preview](https://deploy-preview-2951--keen-mayer-a86c8b.netlify.app/foundation/distinctive-brand-assets)

#### What does this PR do?

Adds a page speaking to "How to implement HPE's distinctive brand assets"

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### What are the relevant issues?

Closes #2953

### CODE REVIEW

File: aries-site/src/pages/foundation/distinctive-brand-assets.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+HPE fonts are provided by the HPE theme and Figma. [Visit typography](/foundation/typography) for 
+more detail.
+
+### Call to action button
+Call to action buttons provide an opportunity to make the HPE Element a distinctive and 
+recognizable feature associated with key value delivered by HPE’s edge-to-cloud promise.
+
+CTA buttons are not a replacement for primary action buttons. They have a distinct 
+purpose and styling. For details on when to use one versus the other, read the 
+[CTA vs. primary action buttons](/components/button#).
```
Comment (taysea) [inline aries-site/src/pages/foundation/distinctive-brand-assets.mdx:34]:
```suggestion
[CTA vs. primary action buttons](/components/button#when-to-use-a-cta-button).
```

File: aries-site/src/data/structures/foundation.js [hunk 2]
```diff
@@ -65,6 +67,36 @@ export const foundation = [
...
+    components that can be added.`,
+    preview: {
+      component: () => <Box fill background="datawave-multi-5" />,
+    },
+    seoDescription: `To be easy to mind and easy to buy as the market leader
+    in edge to cloud, we must be consistent and fresh. Our distinctive 
+    brand assets are comprised of the top elements that are always used, 
+    along with several optional components that can be added. Use "always" 
+    assets combined with the "flexible" brand assets to make recipes.`,
+    sections: [],
```
Comment (taysea) [inline aries-site/src/data/structures/foundation.js:85]: Should we add in the sections? Does that map into the search results at all?

**Review by taysea — COMMENTED** [review]
Tiny comment, otherwise the content looks good. Tagging some others for review.

**Comment by taysea** [comment]
@SOjaHPE agree to showing a visual of the GreenLake badge (name can't change bc that's what Brand Central calls it).

wrt to updating the Global Header page, I think that falls under separate scope of work tracked here. Wondering if there's any clarifier/disclaimer we might be able to add to that page in the meantime though: https://github.com/grommet/hpe-design-system/issues/2956

@halocline Do you want to take care of adding the GreenLake badge visual to the assets page?

**Comment by halocline** [comment]
> @halocline Do you want to take care of adding the GreenLake badge visual to the assets page?

@SOjaHPE , @taysea -- I've included an image and added additional copy.

**Pull request merged**